en/peripherals/dshot.md

```diff
@@ -0,0 +1,78 @@
+# DShot
+
+DShot is an alternative ESC protocol to traditional PWM or OneShot.
+It has several advantages compared to those:
+- Reduced latency
+- Checksum (increased robustness)
+- Digital encoding (no need for ESC calibrations)
+- Possibility to get feedback (telemetry)
+- Sending commands (such as reversing the motor spin direction)
+
+ESC's are connected and wired the same way as for [PWM Outputs](pwm_escs_and_servo.md).
+It is possible to switch between both protocols by changing software parameters only.
+ESC's automatically detect the protocol on startup.
+
+## Configuration
+DShot comes with different speed options: *DShot150*, *DShot300*, *DShot600* and *DShot1200*.
+It defines the speed in number of bits per second.
+Select the highest speed supported by the ESC, which can be found in the datasheet.
+
+> **Warning** As usual: change ESC configuration parameters without props mounted.
+
+Enable DShot with the [DSHOT_CONFIG](../advanced_config/parameter_reference.md#DSHOT_CONFIG) parameter by selecting the appropriate speed and reboot the vehicle.
+
+> **Note** If the parameter does not exist, the board does not support DShot.
+
+<span></span>
+
+> **Tip** On boards with an IO, DShot is only supported on the AUX pins.
+> It is best to disable the IO via [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and then connect the motors to the AUX pins instead of MAIN.
+
+If connected via battery, the ESC's should initialize and the motors turn when armed.
+In case the motors do not spin at lowest throttle, increase [DSHOT_MIN](../advanced_config/parameter_reference.md#DSHOT_MIN) until they spin.
+
+## Sending commands
```

Comment:
Link to module docs for dshot when they exist? 
I'd be tempted to push these examples into dshot module docs along with a link to this doc. Then have a very much reduced section here.